SimTracker/SiStripDigitizer/plugins/PreMixingSiStripWorker.cc

```diff
@@ -65,6 +65,15 @@ class PreMixingSiStripWorker : public PreMixingWorker {
   edm::InputTag SiStripAPVPileInputTag_;
   std::string SistripAPVListDM_;  // output tag
 
+  edm::ESGetToken<TrackerGeometry, TrackerDigiGeometryRecord> const pDDToken_;
+  edm::ESGetToken<SiStripBadStrip, SiStripBadChannelRcd> const deadChannelToken_;
+  edm::ESGetToken<SiStripGain, SiStripGainSimRcd> const gainToken_;
+  edm::ESGetToken<SiStripNoises, SiStripNoisesRcd> const noiseToken_;
+  edm::ESGetToken<SiStripThreshold, SiStripThresholdRcd> const thresholdToken_;
+  //edm::ESGetToken<SiStripPedestals, SiStripPedestalsRcd> const pedestalToken_;
+  edm::ESGetToken<SiStripApvSimulationParameters, TrackerTopologyRcd> apvSimulationParametersToken_;
```

Comment:
```suggestion
  edm::ESGetToken<SiStripApvSimulationParameters, SiStripApvSimulationParametersRcd> apvSimulationParametersToken_;
```
this explains the exceptions in the relval tests [[1](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-2430a0/14336/runTheMatrixINPUT-results/250202.2_TTbar_13+TTbar_13INPUT+DIGIPRMXUP15APVSimu_PU25+RECOPRMXUP15_PU25+HARVESTUP15_PU25/step2_TTbar_13+TTbar_13INPUT+DIGIPRMXUP15APVSimu_PU25+RECOPRMXUP15_PU25+HARVESTUP15_PU25.log)], [[2](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-2430a0/14336/runTheMatrixINPUT-results/250202.4_TTbar_13+TTbar_13INPUT+DIGIPRMXUP15APVSimu_PU25+RECOPRMXUP15_HIPM_PU25+HARVESTUP15_PU25/step2_TTbar_13+TTbar_13INPUT+DIGIPRMXUP15APVSimu_PU25+RECOPRMXUP15_HIPM_PU25+HARVESTUP15_PU25.log)]

```
----- Begin Fatal Exception 20-Apr-2021 01:57:53 CEST-----------------------
An exception of category 'NoProxyException' occurred while
   [0] Processing  Event run: 1 lumi: 37 event: 1804 stream: 0
   [1] Running path 'HLTAnalyzerEndpath'
   [2] Prefetching for module L1TRawToDigi/'hltGtStage2Digis'
   [3] Prefetching for module RawDataCollectorByLabel/'rawDataCollector'
   [4] Prefetching for module SiStripDigiToRawModule/'SiStripDigiToRaw'
   [5] Calling method for module PreMixingModule/'mixData'
Exception Message:
No data of type "SiStripApvSimulationParameters" with label "" in record "TrackerTopologyRcd"
 Please add an ESSource or ESProducer to your job which can deliver this data.
----- End Fatal Exception -------------------------------------------------
```

Reply:
Thanks @mmusich! Fixed.